OoyalaAPISampleApp/app/src/main/java/com/ooyala/sample/lists/OoyalaAPIListActivity.java

```diff
@@ -39,6 +40,7 @@ public void onCreate(Bundle savedInstanceState) {
     selectionMap = new LinkedHashMap<String, PlayerSelectionOption>();
     //Populate the embed map
     selectionMap.put("ContentTree for Channel", new PlayerSelectionOption(CHANNEL_CODE, "c0cTkxOqALQviQIGAHWY5hP0q9gU", "http://www.ooyala.com", ChannelContentTreePlayerActivity.class) );
+    selectionMap.put("ContentTree for Playlists", new PlayerSelectionOption(CHANNEL_CODE, "c0cTkxOqALQviQIGAHWY5hP0q9gU", "http://www.ooyala.com", PlaylistListActivity.class) );
```

Comment:
do we use the CHANNEL_CODE at all? I don't see it being used. Pass the PCODE instead?